node/lib/ILogger.ts

```diff
@@ -1,8 +1,11 @@
 export interface ILogger {
-    error(msg: string, logObject?: any): void;
-    warn(msg: string, logObject?: any): void;
-    info(msg: string, logObject?: any): void;
-    verbose(msg: string, logObject?: any): void;
-    debug(msg: string, logObject?: any): void;
+    // Example of loggingMetadata is correlationVector
+    error(msg: string, logObject?: any, loggingMetadata?: string): void; 
```

Comment:
>  error(msg: string, logObject?: any, loggingMetadata?: string): void;  [](start = 2, length = 71)

How does this work usage-wise, if i have no logObject but loggingMetadata? error("myMessage", "my-CoVector")